clang/test/CXX/drs/cwg1xx.cpp

```diff
@@ -119,6 +119,16 @@ namespace cwg109 { // cwg109: yes
   };
 }
 
+namespace cwg110 { // cwg110: 2.8
+template <typename>
+void f();
+
+class f;
+
+template <typename>
+void f(int);
+} // namespace cwg110
```

Comment:
It'd be nice to also check that `class f` works here, and that uses of plain `f` name the function template.

Reply:
I updated the test. Does it look good now?